benchmark/BDN.benchmark/Network/NetworkBase.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+using System.Collections.Concurrent;
```

Comment:
make sure to remove packages that are not being used

Reply:
done